frontend/src/routes/paths.js

```diff
@@ -12,6 +12,7 @@ const paths = {
   confirm: "/confirm",
   healthMeasurements: "/measurements",
   myStory: "/mystory",
+  resources: "/resources",
```

Comment:
Please delete the code related `resources` site. It should not appear in this feature.

Reply:
Deleted